src/dynamo/models/content.ts

```diff
@@ -262,7 +262,7 @@ const content = {
         await putBody(document.id, body);
       }),
     );
-    client.batchPut(docs);
+    client.batchPut(docs.map((doc) => doc.document));
```

Comment:
docs is temporary object that was added when refactoring to TypeScript, that contains the document to write to dynamo plus the body to write to S3. The batchPut to dynamo was using the temp documents instead of the desired part